client/src/components/ElementActions/PublishAction.js

```diff
@@ -3,6 +3,7 @@ import { compose } from 'redux';
 import AbstractAction from 'components/ElementActions/AbstractAction';
 import publishBlockMutation from 'state/editor/publishBlockMutation';
 import i18n from 'i18n';
+import jQuery from 'jquery';
```

Comment:
I think we discussed in Slack, you could probably get this from `window.jQuery` if you wanted to keep it a bit slimmer